### ISSUE TITLE
Playlist display overflows

### ISSUE THREAD

**Issue by SanketDG** [mps-youtube/mps-youtube#722]
https://asciinema.org/a/xN5TyyOhDrCWRyW3VwZq4HxVi

**Comment by Razesdark** [comment]
Try running `LC_ALL= mpsyt`(notice the space between *=* and *mpsyt* and see if that fixes the problem

**Comment by SanketDG** [comment]
`LC_ALL` is reponsible for the language right? How it would help the overflow problem?

Nonetheless, the command doesnt seem to fix anything

**Comment by Razesdark** [comment]
Yes, but mpsyt sets locale based on LC_ALL if set. Overriding LC_ALL to "" worked for me. You could try also setting `LC_DATE=` and see if that fixes the problem.

Alternatively try to comment out the line 
```python
locale.setlocale(locale.LC_ALL, "")  # for date formatting
```
on line *45* in *main.py*

We're not handling locales very good atm

**Comment by SanketDG** [comment]
Seems like the commenting out worked.

But that strange, `mpsyt` sets the locale which leads to breaking the display?

Also, since it just a matter of "length" when it comes to different display formats, why not just offset that before displaying?

**Comment by Razesdark** [comment]
I think the current code assumes that we would be able to create a fixed length DD.MM.YYYY or MM.DD.YYYY format for each respective locale, but as shown with in this case it doesn't actually adhere to this assumption and therefore breaks the view.

Another case of this is with chinese/japanese locales, where mixes of full width and half width characters often mess up spacing as well, as they are in YYYY年MM月DD日 format.

Just adjusting length of the column to allow for longer dates is less than ideal in my opinion, as I think it takes way too much space. If we should allow longer date forms than than the current 10 digit variants it should be optional behavior. I suggest altering the dates to be either a `DD-MM-YYYY`, `MM-DD-YYYY` or a `YYYY-MM-DD` format depending on the users locale.

**Comment by SanketDG** [comment]
Bump.

So, I assume the ideal solution is to display in `DD.MM.YYYY` and other variants?

Currently the date format is very long with day and long month format, its not ideal at all.

Do we need to depend on the locale at all?

Also, curious how the song view doesnt break the playlist view breaks
